test/ipc_devdax_prov_consumer.c

```diff
@@ -36,7 +36,9 @@ int main(int argc, char *argv[]) {
     umf_devdax_memory_provider_params_t devdax_params =
         umfDevDaxMemoryProviderParamsDefault(
             getenv("UMF_TESTS_DEVDAX_PATH"),
-            atol(getenv("UMF_TESTS_DEVDAX_SIZE")));
+            atol(getenv("UMF_TESTS_DEVDAX_SIZE")
```

Comment:
can't we use `size` already set and checked above...? (same for `path`

Reply:
Right :-D I must have been blind ;-)
Done.